web/src/main/webapp/my-app/layout/controllers.js

```diff
@@ -157,7 +157,7 @@ define(['angular', 'jquery'], function(angular, $) {
           var index = $scope.$parent.layout.indexOf(result[0]);
 
           // Remove from layout
-          $scope.$parent.layout.splice(index, 1);
+          $scope.$apply($scope.$parent.layout.splice(index, 1));
```

Comment:
:question: is https://github.com/uPortal-Project/uportal-home/pull/746 intentionally or accidentally mixed here?

Reply:
A little of both. I mistakenly branched off of that branch, did some work, realized what I'd done, and for the sake of my sanity (since the changes in #746 are minor and will likely be merged today) decided to leave it alone. I will mark this branch WIP until #746 is merged, then update it.